hummingbot/strategy/strategy_base.pyx

```diff
@@ -138,6 +138,9 @@ cdef class StrategyBase(TimeIterator):
     def order_tracker(self) -> OrderTracker:
         return self._sb_order_tracker
 
+    def limit_order_from_order_tracker(self, order_id: str):
+        return self.order_tracker.get
+
```

Comment:
Not quite sure what this is for?

Reply:
This was a change that should have never been included in the branch. Thank you @CrimsonJacket, good catch. I have removed it.